tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -798,20 +794,20 @@ TEST(caffe2, importClip) {
     updateInputPlaceholdersByName(ctx, &mod, {"inputs_0"}, {&inputs_0});
   }
 
-  auto *res = ctx.get(output);
-  EE.compile(CompilationMode::Infer, F);
-  EE.run(ctx);
-
-  auto result = res->getHandle();
-  std::vector<size_t> expectedDims = {5, 5};
-  std::vector<float> expectedValues = {45.0, 20.0, 59.0, 60.0, 48.0, 20.0, 44.0,
-                                       46.0, 60.0, 28.0, 20.0, 60.0, 20.0, 20.0,
-                                       60.0, 24.0, 37.0, 60.0, 20.0, 60.0, 36.0,
-                                       38.0, 30.0, 60.0, 40.0};
-  EXPECT_TRUE(result.dims().vec() == expectedDims);
-  for (size_t i = 0; i < 5 * 5; i++) {
-    EXPECT_FLOAT_EQ(result.raw(i), expectedValues[i]);
-  }
+  EXPECT_EQ(F->getNodes().size(), 5);
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *minNode = llvm::dyn_cast<MinNode>(saveNode->getInput().getNode());
+  ASSERT_TRUE(minNode);
+  auto *maxNode = llvm::dyn_cast<MaxNode>(minNode->getLHS().getNode());
+  ASSERT_TRUE(maxNode);
+  auto *maxSplatNode = llvm::dyn_cast<SplatNode>(minNode->getRHS().getNode());
```

Comment:
Can you also check that the input to this node is the same as the protobuf? You can access it using `maxSplatNode->getValue()`.

Reply:
@SplitInfinity  the protobuf means the input from  /glow/tests/models/caffe2Models/clip_op_net.pbtxt ?